victorops-twilio.js

```diff
@@ -945,6 +959,31 @@ function leaveAMessage (twiml, context, event, payload) {
           )
         }
       );
+    // If the no voicemail flag is set then we want to play the no voicemail message
+    // and still create an incident in VO with the caller's phone number
+    } else if (NO_VOICEMAIL.toLowerCase() === 'true') {
+      let message = messages.noVoicemailAfterCall(teamsArray[0].name);
```

Comment:
```suggestion
      let message = messages.voicemail(teamsArray[0].name);
      
      if (goToVM !== true) {
        message = `${messages.noAnswer} ${message}`;
      }
```

And then we could delete `messages.noVoicemail` and truncate `messages.noVoicemailAfterCall`